### PR TITLE
Commenting functions

### PR DESCRIPTION
**Pull request by mcrowson**
Added comments to functions that were lacking them (and weren't simply getters/setters) in an attempt to better understand code base.

### CODE REVIEW

File: source/ShowScenes.brs [hunk 1]
```diff
@@ -530,6 +538,7 @@ sub ShowTVSeasonEpisodes(series, season)
 end sub
 
 sub ShowCollections(library)
+  ' TV Show Collections List
```
Comment (bisby) [inline source/ShowScenes.brs]: Collections I believe are for movies, rather than TV shows.

Comment (mcrowson) [reply source/ShowScenes.brs]: I can add TV Shows to collections as well. Is this function the logic behind adding both TV Shows and Movies to collections?

Comment (bisby) [reply source/ShowScenes.brs]: That I was not aware of. This function currently loads collection items as if they were Movie objects. Though I guess abstracting that and loading objects based on their self reported type would be smarter in the long run.

Collections on my server are not currently working (even in the webapp) anyway, so I havent been able to test in a while.

Comment (mcrowson) [reply source/ShowScenes.brs]: Just made a new collection and added a TV Show and Movie. Seems working on 10.3.1 via Docker.

**Review by bisby — APPROVED** [review]

**Pull request merged**